components/knowledge-base-manager.tsx

```
@@ -0,0 +1,309 @@
+                <CardDescription className="text-xs">
+                  {article.categoryName}
+                </CardDescription>
+              </CardHeader>
+              <CardContent className="pt-0">
+                <p className="text-xs text-muted-foreground line-clamp-2">
+                  {article.summary || article.content.substring(0, 100) + '...'}
+                </p>
+                {article.tags && (
+                  <div className="flex flex-wrap gap-1 mt-2">
+                    {article.tags.split(',').slice(0, 2).map((tag, index) => (
+                      <Badge key={index} variant="secondary" className="text-xs">
```

Biome (1.9.4)

[error] Avoid using the index of an array as key property in an element.

This is the source of the key value.

The order of the items may change, and this also affects performances and component state.
Check the React documentation. 

(lint/suspicious/noArrayIndexKey)